### PR TITLE
Add functionality for checking a p4-page-type filter by clicking its la…

### PR DESCRIPTION
**Pull request by angtheod**
…bel inside a result item.

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -43,6 +43,13 @@ $(document).ready(function() {
 		}
 	});
 
+	// Add filter by clicking on the page type label inside a result item.
+	$(".search-result-item-head").off("click").on("click", function () {
+		//$("input[name^='f[']").prop("checked", false);
+		$(".custom-control-input[value=" + $(this).data("term_id") + "]").prop("checked", true);
+		$("#search_form").submit();
+	});
+
```
Comment (angtheod) [inline assets/js/main.js]: @ashish173 
Tagging you to see if any of these changes need to go into presentation layer.

File: templates/tease-search.twig [hunk 2]
```diff
@@ -34,7 +34,7 @@
 				<span class="search-result-item-title">{{ posts_data[post.ID].content_type_text }}</span>
 
 				{% for page_type in posts_data[post.ID].page_types %}
-					<a href="#" class="search-result-item-head">{{ page_type.name }}</a>
+					<button class="search-result-item-head" data-term_id="{{ page_type.term_id }}">{{ page_type.name }}</button>
```
Comment (angtheod) [inline templates/tease-search.twig]: @ashish173 
Changed this to button and had to add some css for it to look again the same.

File: style.css [hunk 3]
```diff
@@ -1681,6 +1681,8 @@ body.brown-bg { background: #FAF7EC; }
 .bold { font-weight: bold; }
 .activefilter-tag { background-color: transparent; text-align: left; cursor: pointer; }
 .clearall { cursor: pointer; text-transform: uppercase; }
+button.search-result-item-head { background-color: transparent; border: none; cursor: pointer; padding: 0; color: #0E538E; font-weight: bold; text-decoration: none; margin-right: 30px; position: relative; font-family: "Roboto", sans-serif; }
+button.search-result-item-head:after { content: ""; position: absolute; top: 0; right: 0; width: 8px; height: 8px; border-radius: 5px; background-color: #0E538E; margin-right: -20px; margin-top: 7px !important; }
```
Comment (angtheod) [inline style.css]: @ashish173 
Some css added for the button.

**Comment by koyan** [comment]
Looks good 👍

**Pull request merged**